index.html

```diff
@@ -101,6 +86,38 @@ <h4 class="about-me-title">A Little Bit About Me...</h4>
             </div>
         </div>
     </main>
-    <!--<main id="contact"></main>-->
+    <main id="contact">
+        <h4 class="title brand">Get in Contact</h4>
+        <div class="contact-container">
+            <div class="contact-area">
```

Comment:
You could probably just add these styles to the <a> tag within, and ditch these div's all together?

Reply:
Alternatively, you can drop the 'contact area' class and target it with '.contact-container>div' (or if you get rid of the divs, '.contact-area>a')